tools/bioformats2raw/.shed.yml

```
@@ -0,0 +1,9 @@
+categories:
+  - Imaging
+  - Convert Formats
+description: Convert image to OME-Zarr
+long_description: Convert image to OME-Zarr using bioformats2raw
+name: bioformats2raw
+owner: Euro-BioImaging
```

```suggestion
owner: euro-bioimaging
```
This makes linter fail: https://github.com/BMCV/galaxy-image-analysis/actions/runs/6736707920/job/18342508002?pr=72#step:5:372